lib/features/app_streak/data/streak_start_repo.dart

```diff
@@ -0,0 +1,20 @@
+import "dart:async";
+
+import "package:riverpod_annotation/riverpod_annotation.dart";
+
+import "../../../utils/timestamp.dart";
+import "timstamp_repo_mixin.dart";
+
+part "streak_start_repo.g.dart";
+
+@riverpod
+class StreakStartRepository extends _$StreakStartRepository
+    with TimestampRepository {
+  @override
+  final storeKey = "__app_streak_counter__streak_start_stamp";
```

Comment:
What about parametrizing it in the config file?

Reply:
sure